tests/config_tree/test_basic_functionality.py

```diff
@@ -439,6 +440,31 @@ def test_to_dict_yaml(test_spec):
     assert yaml_config == config.to_dict()
 
 
+@pytest.mark.xfail(reason="Not yet implemented")
+def test_equals():
```

Comment:
Rather than xfailing, if we don't support equality operations on ConfigTrees we should override the `__eq__` function and throw a `NotImplementedError`.